### ISSUE TITLE
Visibilty of inventory-variables in local play

### ISSUE THREAD

**Issue by jarmoni** [ansible/ansible#14768]
##### Issue Type:
- Bug Report
##### Ansible Version:

```
ansible 2.0.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### Ansible Configuration:

default, without modifications
##### Environment:

Ubuntu 15.10
##### Summary:

Variables declared in group_vars/<inventory_name> are not evaluated in plays executed on localhost
##### Steps To Reproduce:

File group_vars/development:

```
second_dns: 172.168.0.1
```

Simple Playbook:

```
---                                                                                                                                                                                                                                          
- name: just a test
  hosts: some_remote_host
  tasks:
  - debug: msg={{second_dns}}

- name: just a 2nd test
  hosts: 127.0.0.1
  connection: local
  tasks:
  - debug: msg={{second_dns}}
```

Playbook is executed with:

```
ansible-playbook -v -i development
```
##### Expected Results:

I expect, that the value of var 'second_dns' will be printed out twice because inventory-file is given and variable is declared in the according group_vars-file
##### Actual Results:

The first (remote) play is executed correctely. In the second (local) play the variable cannot be evaluated:

```
PLAY [just a test] *************************************************************

TASK [setup] *******************************************************************
ok: [some_remote_host]

TASK [debug] *******************************************************************
ok: [some_remote_host] => {
    "msg": "172.168.0.1"
}

PLAY [just a 2nd test] *********************************************************

TASK [setup] *******************************************************************
ok: [127.0.0.1]

TASK [debug] *******************************************************************
fatal: [127.0.0.1]: FAILED! => {"failed": true, "msg": "'second_dns' is undefined"}

```

**Comment by dagwieers** [comment]
I don't think your host 127.0.0.1 is part of the group `development`. So it doesn't have this variable set.

If you want to set a global variable (for all hosts) you have to define it for group `all` in _group_vars/all_.

**Comment by jarmoni** [comment]
I added 127.0.0.1 to inventory-file `development`. But the variable is still "undefined".
I also tried the following:

```
localhost ansible_connection=local
```

Same result.

Declaration in _group_vars/all_ works but that does not solve my issue because the value of the variable should be different in each environment (inventory).

Is there no solution to parametrize a local task  in dependence of the current inventory?
